### PR TITLE
Updated Trac and GitHub issue references

### LINKED ISSUE #433: Update Trac ticket references in the code
## Are you reporting a feature or a bug?

Task

## Provide detailed reproduction steps (if any)

After 4.7.0 we moved to GitHub, so we need a way to reference GitHub tickets in the code.

We want to use simply `#<number>` references, as we did with Trac, but we need to update exisitng ticket references in the code first - so that there are no collisions.

What we simply want to do is to replace all the Trac `#<number>` issues with full links: `https://dev.ckeditor.com/ticket/<number>`.

### PR DESCRIPTION
**Pull request by mlewand**
## What is the purpose of this pull request?

Maintenance Task

## What changes did you make?

Replaced existing `#<number>` references with permalinks to Trac. Also updated tags in manual tests, from format `<number>` to `trac<number>`.

GitHub references has been updated from `gh<number>` to `<number>` in manual tests, and `#<number>` in js files.

Closes #433.

This issue does not require changelog entry.

### CODE REVIEW

File: tests/plugins/indent/indent.js [hunk 1]
```diff
@@ -174,7 +174,7 @@
 			t.s( 2, 0 );
 		},
 
-		// This might be similar to #2
+		// This might be similar to http://dev.ckeditor.com/ticket/2
```
Comment (Comandeer) [inline tests/plugins/indent/indent.js]: It does not seem to be ticket number.

File: tests/plugins/pastefromword/generated/_lib/q.js [hunk 2]
```diff
@@ -372,7 +372,7 @@ var STACK_JUMP_SEPARATOR = "From previous event:";
 
 function makeStackTraceLong(error, promise) {
     // If possible, transform the error stack trace by removing Node and Q
-    // cruft, then concatenating with the stack trace of `promise`. See #57.
+    // cruft, then concatenating with the stack trace of `promise`. See http://dev.ckeditor.com/ticket/57.
```
Comment (Comandeer) [inline tests/plugins/pastefromword/generated/_lib/q.js]: It does not seem to be ticket number.

File: tests/plugins/tableselection/keyboard.js [hunk 3]
```diff
@@ -174,7 +174,7 @@
 			} );
 		},
 
-		// #415
+		// http://dev.ckeditor.com/ticket/415
```
Comment (Comandeer) [inline tests/plugins/tableselection/keyboard.js]: It's GH ticket number

File: tests/plugins/tableselection/manual/blur.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 @bender-ui: collapsed
-@bender-tags: tc, 16755, 4.7.0, tp1577
+@bender-tags: tc, 16755, 4.7.0, tptrac1577
```
Comment (Comandeer) [inline tests/plugins/tableselection/manual/blur.md]: It's tp ticket number, not trac.

Comment (Comandeer) [inline tests/plugins/tableselection/manual/blur.md]: Additionaly trac ticket is not changed.

File: tests/plugins/tableselection/manual/integrations/clipboard/merge.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
 @bender-ui: collapsed
-@bender-tags: tc, 16755, 4.7.0, tp1833
+@bender-tags: tc, 16755, 4.7.0, tptrac1833
```
Comment (Comandeer) [inline tests/plugins/tableselection/manual/integrations/clipboard/merge.md]: It's tp ticket number, not trac. Additionaly trac ticket is not changed.

File: tests/plugins/tableselection/manual/integrations/clipboard/mergeemptycell.md [hunk 6]
```diff
@@ -1,5 +1,5 @@
 @bender-ui: collapsed
-@bender-tags: tc, 16755, 4.7.0, tp1833
+@bender-tags: tc, 16755, 4.7.0, tptrac1833
```
Comment (Comandeer) [inline tests/plugins/tableselection/manual/integrations/clipboard/mergeemptycell.md]: It's tp ticket number, not trac.

File: tests/plugins/tableselection/manual/integrations/clipboard/nestedtable.md [hunk 7]
```diff
@@ -1,5 +1,5 @@
 @bender-ui: collapsed
-@bender-tags: tc, 16755, 4.7.0, tp1867
+@bender-tags: tc, 16755, 4.7.0, tptrac1867
```
Comment (Comandeer) [inline tests/plugins/tableselection/manual/integrations/clipboard/nestedtable.md]: It's tp ticket number, not trac.

File: tests/plugins/tableselection/manual/integrations/clipboard/nestedtablemultiple.md [hunk 8]
```diff
@@ -1,5 +1,5 @@
 @bender-ui: collapsed
-@bender-tags: tc, 16755, 4.7.0, tp1905
+@bender-tags: tc, 16755, 4.7.0, tptrac1905
```
Comment (Comandeer) [inline tests/plugins/tableselection/manual/integrations/clipboard/nestedtablemultiple.md]: It's tp ticket number, not trac.

File: tests/plugins/tableselection/manual/integrations/link/link.md [hunk 9]
```diff
@@ -1,5 +1,5 @@
 @bender-ui: collapsed
-@bender-tags: tc, 16755, 4.7.0, tp1723
+@bender-tags: tc, 16755, 4.7.0, tptrac1723
```
Comment (Comandeer) [inline tests/plugins/tableselection/manual/integrations/link/link.md]: It's tp ticket number, not trac.

File: tests/plugins/tableselection/manual/integrations/selectall/selectall.md [hunk 10]
```diff
@@ -1,5 +1,5 @@
 @bender-ui: collapsed
-@bender-tags: tc, 16755, 4.7.0, tp1723
+@bender-tags: tc, 16755, 4.7.0, tptrac1723
```
Comment (Comandeer) [inline tests/plugins/tableselection/manual/integrations/selectall/selectall.md]: It's tp ticket number, not trac.

File: tests/plugins/tableselection/manual/integrations/selectall/table.md [hunk 11]
```diff
@@ -1,5 +1,5 @@
 @bender-ui: collapsed
-@bender-tags: tc, 16755, 4.7.0, tp1721
+@bender-tags: tc, 16755, 4.7.0, tptrac1721
```
Comment (Comandeer) [inline tests/plugins/tableselection/manual/integrations/selectall/table.md]: It's tp ticket number, not trac.

File: tests/plugins/tableselection/manual/integrations/selectall/selectall.md (hunk 10 above)
Comment (Comandeer) [inline tests/plugins/tableselection/manual/integrations/selectall/selectall.md]: Additionally trac ticket is not changed.

File: tests/plugins/tableselection/manual/integrations/selectall/table.md (hunk 11 above)
Comment (Comandeer) [inline tests/plugins/tableselection/manual/integrations/selectall/table.md]: Additionally trac ticket is not changed.

File: tests/plugins/tableselection/manual/integrations/tabletools/celldelete.md [hunk 12]
```diff
@@ -1,5 +1,5 @@
 @bender-ui: collapsed
-@bender-tags: tc, 16755, 4.7.0, tp1892
+@bender-tags: tc, 16755, 4.7.0, tptrac1892
```
Comment (Comandeer) [inline tests/plugins/tableselection/manual/integrations/tabletools/celldelete.md]: It's tp ticket number, not trac. Additionally trac ticket is not changed.

File: tests/plugins/tableselection/manual/integrations/tabletools/cellmerge.md [hunk 13]
```diff
@@ -1,5 +1,5 @@
 @bender-ui: collapsed
-@bender-tags: tc, 16755, 4.7.0, tp1574
+@bender-tags: tc, 16755, 4.7.0, tptrac1574
```
Comment (Comandeer) [inline tests/plugins/tableselection/manual/integrations/tabletools/cellmerge.md]: It's tp ticket number, not trac. Additionally trac ticket is not changed.

File: tests/plugins/tableselection/manual/native.md [hunk 14]
```diff
@@ -1,5 +1,5 @@
 @bender-ui: collapsed
-@bender-tags: tc, 16755, 4.7.0, tp1721
+@bender-tags: tc, 16755, 4.7.0, tptrac1721
```
Comment (Comandeer) [inline tests/plugins/tableselection/manual/native.md]: It's tp ticket number, not trac. Additionally trac ticket is not changed.

File: tests/plugins/tableselection/manual/outsideeditor.md [hunk 15]
```diff
@@ -1,5 +1,5 @@
 @bender-ui: collapsed
-@bender-tags: tc, 16755, 4.7.0, tp1721
+@bender-tags: tc, 16755, 4.7.0, tptrac1721
```
Comment (Comandeer) [inline tests/plugins/tableselection/manual/outsideeditor.md]: It's tp ticket number, not trac. Additionally trac ticket is not changed.

File: tests/plugins/tableselection/manual/outsidetable.md [hunk 16]
```diff
@@ -1,5 +1,5 @@
 @bender-ui: collapsed
-@bender-tags: tc, 16755, 4.7.0, tp1579
+@bender-tags: tc, 16755, 4.7.0, tptrac1579
```
Comment (Comandeer) [inline tests/plugins/tableselection/manual/outsidetable.md]: It's tp ticket number, not trac. Additionally trac ticket is not changed.

File: tests/plugins/widgetselection/manual/image2.md [hunk 17]
```diff
@@ -1,4 +1,4 @@
-@bender-tags: 4.6.1, tc, 11064, widgetselection
+@bender-tags: 4.6.1, tc, 1trac1064, widgetselection
```
Comment (Comandeer) [inline tests/plugins/widgetselection/manual/image2.md]: It shouldn't be changed this way.

File: tests/plugins/widgetselection/manual/selectall.md [hunk 18]
```diff
@@ -1,4 +1,4 @@
-@bender-tags: 4.6.1, tc, 11064, widgetselection
+@bender-tags: 4.6.1, tc, 1trac1064, widgetselection
```
Comment (Comandeer) [inline tests/plugins/widgetselection/manual/selectall.md]: This shouldn't be changed this way.

File: tests/tickets/11616/1.md [hunk 19]
```diff
@@ -1,5 +1,5 @@
 @bender-ui: collapsed
-@bender-tags: wyswigarea, tc, 4.5.2, 11616, 9715
+@bender-tags: wyswigarea, tc, 4.5.2, 1trac1616, 9715
```
Comment (Comandeer) [inline tests/tickets/11616/1.md]: This shouldn't be changed this way.

File: tests/tickets/11616/2.md [hunk 20]
```diff
@@ -1,5 +1,5 @@
 @bender-ui: collapsed
-@bender-tags: toolbar, tc, 4.5.2, 11616, 7360
+@bender-tags: toolbar, tc, 4.5.2, 1trac1616, 7360
```
Comment (Comandeer) [inline tests/tickets/11616/2.md]: This shouldn't be changed this way.

File: tests/core/selection/fake.js [hunk 21]
```diff
@@ -114,7 +114,7 @@ bender.test( {
 		if ( !CKEDITOR.env.edge || CKEDITOR.env.version < 14 ) {
 			inputHtml = '<p>[foo <span id="bar">bar</span>]</p>';
 		}
-		
+
```
Comment (mlewand) [inline tests/core/selection/fake.js]: There was a trailing white space, which is not allowed in our standards.

**Comment by mlewand** [comment]
Added requested changes.

File: tests/core/selection/manual/focusafterundoing.md [hunk 22]
```diff
@@ -1,4 +1,4 @@
-@bender-tags: selection, focus, tc, 4.4.6, 12630, 12337
+@bender-tags: selection, focus, tc, 4.4.6, 12630, trac12337
```
Comment (Comandeer) [inline tests/core/selection/manual/focusafterundoing.md]: There is still second ticket number, which was not changed.

File: tests/core/selection/manual/selectionafterfocusing.md [hunk 23]
```diff
@@ -1,4 +1,4 @@
-@bender-tags: selection, focus, tc, 4.4.6, 12630, 12337
+@bender-tags: selection, focus, tc, 4.4.6, 12630, trac12337
```
Comment (Comandeer) [inline tests/core/selection/manual/selectionafterfocusing.md]: There is still second ticket number, which was not changed.

File: tests/plugins/newpage/manual/selectionafternewpage.md [hunk 24]
```diff
@@ -1,4 +1,4 @@
-@bender-tags: selection, focus, tc, 4.4.6, 12630, 12337
+@bender-tags: selection, focus, tc, 4.4.6, 12630, trac12337
```
Comment (Comandeer) [inline tests/plugins/newpage/manual/selectionafternewpage.md]: There is still second ticket number, which was not changed.

File: tests/plugins/tableselection/tableselection.js [hunk 25]
```diff
@@ -20,7 +20,7 @@
 	}
 
 	var tests = {
-		// (#258, #tp2247)
+		// (http://dev.ckeditor.com/ticket/258, #tp2247)
```
Comment (Comandeer) [inline tests/plugins/tableselection/tableselection.js]: It seems like this ticket number is from unknown source… It does not fit to trac or our GH.

Comment (mlewand) [inline tests/plugins/tableselection/tableselection.js]: Since this issue is not available to the public, I decided to remove it all along.

**Comment by mlewand** [comment]
I found a couple extra missing references, fixed in 3d04295e8d057ff526f9cf3440bec40e609fe425.

File: tests/core/selection/manual/focusaftersettingdata.md [hunk 26]
```diff
@@ -1,4 +1,4 @@
-@bender-tags: selection, focus, tc, 4.4.6, 12630, 12337
+@bender-tags: selection, focus, tc, 4.4.6, 12630, trac12337
```
Comment (Comandeer) [inline tests/core/selection/manual/focusaftersettingdata.md]: Still one ticket number left.

**Comment by mlewand** [comment]
Fixed. I did not rebase the branch, so that comments won't lose the reference.

**Comment by mlewand** [comment]
> and fix also the newly added ticket numbers

It's a relatively recent PR, and it's latest common commit with `master` is 83a5ae1508588655b96f8514148cbbb5e86e0dd0.



> But it would be nice to rebase the branch

Done.

**Pull request merged**